Velaptor/InternalExtensionMethods.cs

```diff
@@ -213,7 +213,7 @@ public static void SuppressDisposableTransientWarning<T>(this Container containe
         /// <typeparam name="TService">The interface or base type that can be used to retrieve the instances.</typeparam>
         /// <typeparam name="TImplementation">The concrete type that will be registered.</typeparam>
         /// <param name="container">The container that the registration applies to.</param>
-        /// <param name="suppressDisposal"><see langword="true"/> to ignore dispose warnings if the original code invokes dispose.</param>
+        /// <param name="suppressDisposal"><c>true</c> to ignore dispose warnings if the original code invokes dispose.</param>
         /// <remarks>
         ///     This method uses the container's LifestyleSelectionBehavior to select the exact
```

Comment:
Should the underlined text be spelled without spaces? If so, should it have quotes or be underlined?
```suggestion
        ///     This method uses the container's LifestyleSelectionBehavior to select the exact
```